tensorcircuit/mpscircuit.py

```diff
@@ -8,57 +8,66 @@
 
```

Comment:
many numpy method and numpy attr such as `.shape` in this file. not 100% sure whether these affects jit or backend agnosticity, may require further jit test